frontend/src/metabase/components/DownloadButton.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from "react";
+import ReactDOM from "react-dom";
+
+import Button from "metabase/components/Button.jsx";
+
+export default class DownloadButton extends Component {
+    constructor(props, context) {
+        super(props, context);
```

Comment:
I think the main reason is just legibility. This is totally just personal preference but it feels cleaner to only really reference what you need which is neither in this case.